### ISSUE TITLE
Windows: Pressure sensitivity does only work after second start

### ISSUE THREAD

**Issue by andreasb242** [xournalpp/xournalpp#659]
On the first start pressure sensitivity is not available, after restart it's working.
As this is probably a GTK issue there will probably be no real fix, instead create a workaround.

**Comment by andreasb242** [comment]
After testing I found out:
start Xournal++, do not touch with the pen
start again, touch with the Pen. Does not work
start the 3rd time: working

So it don't matter how many times it's started, it needs pen contact and then a restart. Strange, and really hard to find a workaround.

**Comment by andreasb242** [comment]
GTK Bug https://gitlab.gnome.org/GNOME/gtk/issues/1549

**Comment by cbm755** [comment]
Also, just to note that xournal does this too (presumably b/c its underlying GTK bug).

**Comment by andreasb242** [comment]
Yes, this is a GTK Bug, I found it already, but had no time to fix it.

```
 andreasb242 @andreasb242 commented 1 day ago

I got a hint on my other Merge Request: https://gitlab.gnome.org/GNOME/gtk/blob/gtk-3-24/gdk/win32/gdkdevicemanager-win32.c#L1065

The axes count is initialized dynamically on init. My guess is now, that this count is not there, if the pen has not yet touched the screen. I didn't check this yet, but it would produce exactly the current behavior.
```

I could confirm this behavior, but I didn't get my mac pull merged yet, so I wont work on two GTK pulls at the same time.

**Comment by jason076** [comment]
I have a Surface Pro4 and my Stylus is always recognized as a normal mouse. I have tried the workarounds from above, but they dont't work. I have the Xournal++ Version 1.0.12

**Comment by LittleHuba** [comment]
@jason076 
Please try #1151.

**Comment by jason076** [comment]
> 
> 
> @jason076
> Please try #1151.
> You will have to compile by hand though.
@LittleHuba 

I have compiled and tried it, but it does not recognizes any input devices. The stylus is still working like a normal mouse.

**Comment by LittleHuba** [comment]
Did you activate the new input system in the settings and set the specific device class for your devices?

**Comment by jens1o** [comment]
Is there help I could provide to get this resolved?

**Comment by BUCKFAE** [comment]
> > @jason076
> > Please try #1151.
> > You will have to compile by hand though.
> 
> @LittleHuba
> I have compiled and tried it, but it does not recognizes any input devices. The stylus is still working like a normal mouse.

Same for me. Any updates on this? SurfacePro (2017).

**Comment by hacker1024** [comment]
> I have a Surface Pro4 and my Stylus is always recognized as a normal mouse. I have tried the workarounds from above, but they dont't work. I have the Xournal++ Version 1.0.12

I have this issue too, on the latest 1.0.18. The above PR is merged, but only one device comes up when configuring input devices.

I have a Dell Inspiron 7586, which uses the same pen tech as Surface devices (and is compatible with Surface pens). It works on Linux, but not Windows.

Any progress on this? Should I create a dedicated issue?

**Comment by hacker1024** [comment]
It seems that our issue is caused by #1346.

**Comment by gerroon** [comment]
I just installed 1.0.19 and no pressure witrh Wacom trablet on Win 10 no matter how many times I restart.  This used to work in the 1.0.18 :(

**Comment by STASWIT** [comment]
I have the same issue on ROG Flow X13 which have the same pen tech as Surface.

**Comment by rolandlo** [comment]
Have you tried version [1.1.0-hotfix.1](https://github.com/xournalpp/xournalpp/releases/tag/1.1.0-hotfix.1) ?

**Comment by STASWIT** [comment]
Now I have tried it, it fixed my problem. Thanks.
